src/main/scala/viper/gobra/translator/util/ViperUtil.scala

```diff
@@ -46,4 +46,53 @@ object ViperUtil {
       case _ => Violation.violation(s"expected vpr variable or field access, but got $left")
     }
   }
+
+  /**
+    * TODO: should be removed once the corresponding silver function is fixed.
+    *
```

Comment:
Can you please add the link to the `silver` issue that pertains to this comment? Otherwise, it is hard to know what the issue is and to understand if it has been fixed.

Reply:
I added a silver issue: https://github.com/viperproject/silver/issues/610